openhtf/io/frontend/src/app/utils.ts

```diff
@@ -41,6 +41,7 @@ export class SimplifyStatus extends MapPipe {
     'WAITING_FOR_TEST_START': 'WAITING',
     'STOP_WAIT': 'WAITING',
     'TIMEOUT': 'ERROR',
+    'ABORTED': 'ERROR',
```

Comment:
Yeah, but the frontend color schemes are quite a bit more simplistic. Blue for running normally, red for failing normally, green for passing normally, yellow for something went wrong, and grey for something is down/missing. I agree this could use a cleanup, but I'd like to punt on it for now since it's just an elegance of design thing and not user-affecting.